### PR TITLE
:sparkles: Joke command

### PR DESCRIPTION
**Pull request by Gilbert142**
**Please describe the changes this PR makes and why it should be merged:**
Ports over V4 (Dad) Joke command to indev V5, with a new, working API.

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [ ] This PR changes command functions
  - [ ] This PR changes metadata for the command, updated in `commands.js`
  - [x] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/joke.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	await msg.send({
+		embed: {
+			color: Colors.INFO,
+			title: `Loading times are the biggest joke of all`,
+			description: `We're getting you a better one...`,
+		},
+	});
+
+	try {
+		const res = await fetch.get(APIs.JOKE()).set("Accept", "application/json").toJSON();
```
Comment (vladfrangu) [inline Commands/Public/joke.js]:
```suggestion
		const body = await fetch.get(APIs.JOKE()).set("Accept", "application/json").toJSON().onlyBody();
```

File: Commands/Public/joke.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		const { body } = res;
```
Comment (vladfrangu) [inline Commands/Public/joke.js]: Remove

File: Commands/Public/joke.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+		if (body.status === 200 && body.joke && typeof body.joke === "string") {
```
Comment (vladfrangu) [inline Commands/Public/joke.js:16]: Are you SUUURE its body.status?

File: Commands/Public/joke.js (hunk 1 above)
Comment (vladfrangu) [inline Commands/Public/joke.js]: Also, per their docs
```
If you intend on using the icanhazdadjoke.com API we kindly ask that you set a custom User-Agent header for all requests.

Setting a custom User-Agent header for your code will help us be able to better monitor the usage of the API and identify potential bad actors.

A good user agent should include the name of the library or website that is accessing the API along with a URL/e-email where someone can be contacted regarding the library/website.
```

File: Commands/Public/joke.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+			msg.send({
+				embed: {
+					color: Colors.RESPONSE,
+					description: body.joke.length < 2048 ? body.joke : `${body.joke.substring(0, 2045)}...`,
+				},
+			});
+		}
+	} catch (err) {
```
Comment (vladfrangu) [inline Commands/Public/joke.js:24]: `(err)` binding is useless, remove it

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: Commands/Public/joke.js (hunk 4 above)
Comment (wasdennnoch) [reply Commands/Public/joke.js:24]: Are we not doing logging of errors anymore? 🤔

File: Commands/Public/joke.js (hunk 3 above)
Comment (Gilbert142) [reply Commands/Public/joke.js:16]: Yes, says so in their docs.

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**